test/UnitTest.cs

```diff
@@ -1139,6 +1139,50 @@ public void TestSubqueryRewrite()
                 var result = TU.ExecuteSQL(sql, out phyplan, option); Assert.IsTrue(TU.error_.Contains("one row"));
             }
         } 
+
+        [TestMethod]
+        public void TestSubqueryInFromClause()
+        {
+            var files = Directory.GetFiles(@"../../../../tpch", "*.sql");
+            string scale = "0001";
+            Tpch.CreateTables(true);
+            Tpch.LoadTables(scale);
+            Tpch.AnalyzeTables();
+            string sql = "select c_name,(select count(O.o_orderkey) from orders O where O.o_custkey = 37) as OrderCount from customer C";
+            QueryOption option = new QueryOption();
+            option.optimize_.TurnOnAllOptimizations();
+            option.optimize_.remove_from_ = false;
+            option.explain_.show_output_ = true;
+            option.explain_.show_estCost_ = option.optimize_.use_memo_;
+            option.explain_.mode_ = ExplainMode.explain;
+            var test_result = SQLStatement.ExecSQLList(sql, option);
+            var expect_plan = @"select c_name,(select count(O.o_orderkey) from orders O where O.o_custkey = 37) as OrderCount from customer C
```

Comment:
when you come up with more test cases, put them in expected folder - UnitTest.cs is already too big